src/app/server/java/AndroidAppServerWrapper.h

```diff
@@ -22,6 +22,10 @@
 
 CHIP_ERROR ChipAndroidAppInit(void);
 
+void GetFabricCount(uint8_t & count);
```

Comment:
This does not look like the right place to put these methods - the header is `AndroidAppServerWrapper` and the methods are relating to app server init and shutdown.

Fabric count and list seem completely out of place.